src/rds/filterStartableDBInstances.js

```diff
@@ -0,0 +1,18 @@
+const listAllDBInstances = require('./listAllDBInstances');
+
+function allStartableDBInstances(instance) {
+  return instance.DBInstanceStatus == 'stopped';
+};
+
+module.exports = function filterStartableDBInstances() {
+  return new Promise((resolve, reject) => {
+    listAllDBInstances()
+      .then(data => {
+        resolve(data
+          .DBInstances
+          .filter(allStartableDBInstances)
+          .map(instance => instance.DBInstanceArn));
+      })
+      .catch(err => reject(err));
```

Comment:
No need to wrap in a promise. Can also then remove the catch that seemingly just throw the exception again.